### DISCUSSION TITLE
Preservation Method v2 - CDE 8028962

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#64] [Metadata Field Ratification]
caDSR Link: [5432521](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=5432521%20and%20ver_nr=1.0)

Definition: "Text term that represents the method used to preserve the sample."

**Comment by GeoffLyle** [comment]
This CDE contains Permissible Values **Frozen** and **Snap Frozen**.
Treehouse has internally harmonized using severe cold to preserve tissue variations (ex. 'fresh frozen', 'snap frozen', 'frozen') to **Frozen**.  We are likely to only use the **Frozen** PV and not the **Snap Frozen** PV.

**Comment by GeoffLyle** [comment]
Also note that similar to #50 this CDE does not contain an "Other" or "Not Reported" field.

**Comment by calkinsh** [comment]
This CDE will not work for any of our germline (blood/saliva) samples because it is tissue focused. And because it is does not have an Other or Not Reported field we will have nothing to populate for those (or for a very small number of tissue samples whose preservation method doesnt fit into this list).  We are OK with moving forward as-is but would prefer this not be a required field given the limitations for our dataset

**Comment by hbeale** [comment]
Do we need both frozen and snap frozen? The definition of Frozen does not exclude snap frozen. I suggest removing snap frozen.

**Comment by jknable** [comment]
Considering the CHOP response in #50 that CDE 5432521 was focused on Tissue and did not have adequate terms to address Blood preservation methods, and that CDE 7351590 is a “CDISC” specific CDE which limits the ability to add additional permissible values, we recommend CDE 8028962v2 which is broader in scope.  

We have updated this CDE to include EDTA as a permissible value per CHOP’s use case, please advise if additional values are needed for the CHOP data.  

“Not Reported” and “Unknown” are included in version 2.  Will this support your data that does not map to one of the permissible values, or do you still need the generic null value of “Other”?

**Reply by calkinsh** [reply]
yes this should work well for us - thank you for the suggestion. The majority of our blood specimens are EDTA preserved, so this covers almost all of our use cases. There are a small number of specific Blood part preservation values that still wont be represented (like Serum Tube) but I think that is ok - they are minimal and now that we have values for Not Reported and Other, we have something to fill in here.  For CHOP this CDE is preferable over 5432521.

As a note, we also have a very small number of tissue samples that are noted as preserved in paraffin block but unknown whether they are formalin fixed or not, so we would not want to list them as FFPE. I'm not sure this warrants adding a new value, since it is very rare, but I wanted to highlight it in case other folks have more cases than we do.

**Comment by GeoffLyle** [comment]
caDSR link to CDE [8028962](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=8028962%20and%20ver_nr=2).

**Comment by GeoffLyle** [comment]
CDE accepted at March 14, 2024 meeting.

**Reply by jknable** [reply]
CDE has been added to caDSR under Federation content.
